public/app/features/alerting/unified/components/rule-viewer/AlertRuleMenu.tsx

```diff
@@ -15,7 +15,7 @@ import { createRelativeUrl } from '../../utils/url';
 import { DeclareIncidentMenuItem } from '../bridges/DeclareIncidentButton';
 
 interface Props {
-  promRule: Rule;
+  promRule?: Rule;
```

Comment:
This is odd without any other changes to this component, are we re-using this interface in other places?

Reply:
This is mostly for `RuleActionButtons.V2` to work 🙂